ONSdigital/dp-frontend-filter-dataset-controller
# Release beta 1.13.0

### What

* RCHTTP client updated to latest dp-api-clients-go

### How to review

* Checkout branch
* Ensure filter journey still works

### Who can review

Anyone except me
